pkg/fleet/installer/default_packages.go

```diff
@@ -104,10 +101,28 @@ func apmLanguageEnabled(p Package, e *env.Env) bool {
 	return false
 }
 
+func apmLanguageVersion(p Package, e *env.Env) string {
+	if apmLibVersion, ok := e.ApmLibraries[packageToLanguage(p.Name)]; ok {
+		return apmLibVersion.AsVersionTag()
+		// TODO(paullgdc): Emit a warning here if APM packages are not pinned to at least a major
+	}
+	return "latest"
+}
+
 func packageToLanguage(packageName string) env.ApmLibLanguage {
 	lang, found := strings.CutPrefix(packageName, "datadog-apm-library-")
 	if !found {
 		return ""
 	}
 	return env.ApmLibLanguage(lang)
 }
+
+func agentVersion(_ Package, e *env.Env) string {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is the patch version supposed to be included in the `minor` variable?

Reply:
In theory you could exclude the patch from the minor by leveraging the major / minor version tags on OCI that we'll have thanks to APM's initiative.
They're not there yet as we're lacking a release, but the mechanism for libraries is shared with the agent.